src/third_party/install/SConscript

```diff
@@ -0,0 +1,59 @@
+Import("env")
+Import("rocksdb")
+
+# InjectThirdPartyIncludePaths in a hard way
+env.PrependUnique(CPPPATH=['#/src/third_party/install/include'])
+env.PrependUnique(LIBPATH=['#/src/third_party/install/lib'])
+
+rocks_dir = '#/src/third_party/rocksdb/'
+install_lib = 'lib/'
+rocks_lib_l = 'rocksdb'
+
+def CheckClean(env):
+    if env.GetOption('clean'):
+        env.Execute(Action(clean_rocksdb))
+
+def make_rocksdb_with(make_part):
+    import subprocess
+
+    build_dir = Dir(rocks_dir).get_abspath()
+    p = subprocess.Popen(
+            ['make', 'INSTALL_PATH=../install', 'DEBUG_LEVEL=0'] + make_part,
```

Comment:
Why not propagate `--dbg=on` option from SCons command line?
Thus we will be able to build debug version when n ecessary.